supply-chain-security/compromises/2010/aurora.md

```diff
@@ -0,0 +1,20 @@
+# Operation Aurora
+
+The hackers could have access to software configuration management systems
+(SCM) in many companies including Google and Adobe. This allowed them to steal
+the source code or make stealthy changes in the source of the many products.
+
+The SCM was developed by a company called Perforce. This system has had some
+known vulnerabilities (detected by McAfee).  The attacker most likely exploited
+those security holes to gain unauthorized access to the system.
+
+## Impact
+
+More than 34 organizations affected, including Symanted, Northrop Grumman,
+Morgan Stanley, Dow chemical, Yahoo, Rackspace, Adobe and Google.
+
+## Type of Compromise
```

Comment:
Let's set a good example for caps consistency from [contributing guide](https://github.com/cncf/sig-security/blob/master/CONTRIBUTING.md) for our new additions.. I think we need to add a CI for this. Which unfortunately will probably fail for our existing repo :p